ossrelease/gen_man.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+'''
+Generates the man pages for a given release
+'''
+
+import argparse
+import re
+import shutil
+import subprocess
+import os
+
+# import ossrelease modules
+import conf
+
+def main():
+    '''
+    Run!
+    Prints results to the screen.
+    '''
+    # Parse args and define some basic params
+    opts = conf.get_conf()
+    args = parse_args()
+    doc_dir = opts['SALT_REPO_PATH'] + '/doc/'
```

Comment:
it does not matter if `SALT_REPO_PATH` has a trailing `/` as there is one in `/doc/`. Are you suggestion handling it a different way?

Reply:
You could have a trailing slash in your config settings, since that is defined by the user. If you do, then you'll run into problems with the way this is currently written. Just thinking about future-proofing this.

Reply:
`<example-salt-repo-path>/ + /doc/`